src/schema.ts

```diff
@@ -76,22 +86,33 @@ export const resolvers = {
       return findUser;
     },
 
-    users: async (_, list: { numUsers: number }, context: Context): Promise<UserList> => {
+    users: async (_, list: { data: UserQueryInput }, context: Context): Promise<UserList> => {
       const secret = process.env.JWT_SECRET ?? 'secret';
       jwt.verify(context.token, secret);
 
       const DEFAULT_NUMBER_OF_USERS = 10;
-      const takeUsers = list.numUsers ?? DEFAULT_NUMBER_OF_USERS;
+      const takeUsers = list.data.numUsers ?? DEFAULT_NUMBER_OF_USERS;
+      const skipUsers = list.data.skip ?? 0;
 
       const userRepository = getRepository(User);
-      const usersList = await userRepository.find({ order: { name: 'ASC' }, take: takeUsers });
+      const response = await userRepository.findAndCount({ order: { name: 'ASC' }, skip: skipUsers, take: takeUsers });
+      const usersList = response[0];
+      const totalUsers = response[1];
```

Comment:
Olha esse destruct pra coletar as variáveis, caso vc goste:

```suggestion
      const [usersList, totalUsers] = await userRepository.findAndCount({ order: { name: 'ASC' }, skip: skipUsers, take: takeUsers });
```